backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java

```diff
@@ -29,5 +37,18 @@ public static String resolveContent(Message message, Map<String, String> metadat
 
                     return updatedContent.replace(urlToReplace, entry.getValue());
                 }, (oldValue, newValue) -> newValue);
+
+        try {
+            jsonNode = new ObjectMapper().readTree(resolvedContent);
+        } catch (JsonProcessingException e) {
+            ObjectMapper objectMapper = new ObjectMapper();
+            try {
+                jsonNode = objectMapper.readTree("{\"text\": \"" + content + "\", \"type\": \"text\"}");
+            } catch (JsonProcessingException ex) {
+                ex.printStackTrace();
+            }
```

Comment:
In the ticket we agreed on returning the content as is if we cannot parse it to a string so I would simply `return resolvedContent;` in the `catch` clause.

Reply:
We cannot because the return type of `content` is `JsonNode`

Reply:
This discussion is now on https://github.com/airyhq/airy/issues/934#issuecomment-779759231